Sources/Web3Core/KeystoreManager/EtherscanTransactionChecker.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  EtherscanTransactionChecker.swift
+//  Created by albertopeam on 28/12/22.
+//
+
+import Foundation
+import _Concurrency
```

Comment:
Short research showed, that this one is a private. Why is it here and could we change it to something public instead?

Reply:
Just to mention, I aware that private frameworks are pretty stable as well, but also I'm aware that such apps that is using such within itself could be declined on AppStore review.

Reply:
fixed